ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -267,6 +267,8 @@ ompi_mtl_ofi_component_register(void)
                                     MCA_BASE_VAR_SCOPE_READONLY,
                                     &ompi_mtl_ofi.num_ofi_contexts);
 
+    opal_common_ofi_register_mca_variables(&mca_mtl_ofi_component.super.mtl_version);
```

Comment:
This is actually a comment about the header -- should there be a "Copyright (c)" line added at the top? (I couldn't seem to add a comment where there wasn't a code change.)

Reply:
@hppritcha You have LANL copyrights in the header; I don't know if you need to add Triad copyrights, too.

Reply:
missed that with the cherry-pick of a cherry-pick, updating